src/assets/data/defaultConfig.ts

```diff
@@ -33,6 +35,15 @@ export const DefaultSourceTypeRegistrationBody = {
   sourceTypeProducer: 'RADAR'
 }
 
+// *The icon for On Demand assessments  (REMOTE CONFIG KEY: `on_demand_assessment_icon`)
+export const DefaultOnDemandAssessmentIcon = 'assets/imgs/day-night.png'
```

Comment:
can we keep this simple by default? Like a questionnaire symbol with a + sign?

Reply:
This icon was the requirement for Dynamore. I'll change it to a more general icon, but I'll keep the image file so we can modify this for Dynamore in the remote config.

Reply:
I've changed the icon and updated the screenshots above. Let me know what you think.